routes/index.js

```diff
@@ -76,6 +76,10 @@ var routes = {
     };
     var request_id = request.headers['x-request-id'];
 
+    if (transaction.description === "Thunderbird") {
+      metadata.thunderbird = true;
+    }
```

Comment:
No, but description remains untouched as "Thunderbird". This flag is for the donation email rendering code